packages/space/src/useSpace.ts

```diff
@@ -50,6 +50,19 @@ export const defaultProps = {
     default: false,
   },
 
+  fill: {
+    type: Boolean,
+    default: false,
+  },
+
+  fillPercentage: {
+    type: Number,
+    default: 100,
+    validator: (val: unknown) => {
+      return isNumber(val) && val > 0 && val <= 100
+    },
```

Comment:
I don't think the validator is necessary right here though. Maybe we change change the name from <s> fillPercentage</s> to  `fillRatio`

Reply:
I quite agree with you. `fillRatio` is much better than `fillPercentage`.